src/gui/PageView.h

```diff
@@ -152,6 +152,11 @@ class XojPageView: public Redrawable, public PageListener {
     bool onMotionNotifyEvent(const PositionInputData& pos);
     void onMotionCancelEvent();
 
+    /**
+     * for popup windows like the floating toolbox
+     */
+    bool onButtonClickEvent(const PositionInputData& pos);
```

Comment:
**Documentation:**
If the comment in `StylusInputHandler` on lines 46-60 is applied, this documentation comment should describe the difference between `onButtonClickEvent` and `onButtonPressEvent`, `onButtonReleaseEvent` (e.g. `this event fires even when no input sequence is actively running`).

---

**Design:**

For future programmers trying to add additional click functionality, I think it could be confusing that `onButtonClick` is only called by `StylusInputHandler` (while other events like `onButtonPressEvent` are also used in, for example, `PenInputHandler`).

Do you think it would make sense for `onButtonClickEvent` to also be called in `onButtonReleaseEvent`?
  * If (in StylusInputHandler) we only call `onButtonClickEvent` directly when there's no input sequence running, we won't be double-calling `onButtonClickEvent`.
  * Additionally, this would let us remove some of the duplicated code I commented about in `PageView`.